checks/microbenchmarks/cpu/simd/nsimd.py

```diff
@@ -60,6 +58,10 @@ class NsimdTest(rfm.RegressionTest):
     def set_descr(self):
         self.descr = f'testing {self.bench_name}'
 
+    @run_before('compile')
+    def skip_test(self):
+        self.skip_if(True, 'skipping legacy test (nsimd no longer installed)')
```

Comment:
```suggestion
        self.skip('skipping legacy test (nsimd no longer installed)')
```
We could be checking that the modulefile exists. In any case if we want to keep the message let's update the skipif to simple skip and we can merge it.

Reply:
Maybe it's also better to skip it after init so that we don't create the firectory, what do you think?

Reply:
must be before compile